app/models/deputy.rb

```diff
@@ -8,4 +8,6 @@ class Deputy < ApplicationRecord
 
   has_many :deputy_legislatures
   has_many :legislatures, through: :deputy_legislatures
+
+  enum room: { deputat: 1, senator: 0 }
```

Comment:
We might move it to the top, based on the [Rails Guide](https://rails.rubystyle.guide/#macro-style-methods)